machine-learning/db/DBConnector.py

```diff
@@ -35,6 +36,7 @@ def execute_query(sql_query):
     df_raw : DataFrame
         Pandas DataFrame with raw data resulting from query
     """
+    log(sql_query)
```

Comment:
We could add a simple description to the log message:

_Fetch data from the db with this query:_